docs/standard/base-types/regular-expression-example-scanning-for-hrefs.md

```diff
@@ -33,7 +33,9 @@ The following example searches an input string and displays all the href="…" v
  [!code-vb[RegularExpressions.Examples.HREF#2](../../../samples/snippets/visualbasic/VS_Snippets_CLR/RegularExpressions.Examples.HREF/vb/example.vb#2)]  
   
  The regular expression pattern `href\s*=\s*(?:["'](?<1>[^"']*)["']|(?<1>\S+))` is interpreted as shown in the following table.  
-  
+
+<!-- markdownlint-disable MD011 -->
```

Comment:
The table doesn't render correctly, so this isn't a false positive.

![image](https://user-images.githubusercontent.com/31348972/103649159-aa753a80-4f66-11eb-8e45-98065c409008.png)


One possible fix is to use `<code></code>` instead of backticks, and use the encoding of the vertical bar. Not sure if there is a better solution.

Reply:
It does render correctly. See the live site:

![image](https://user-images.githubusercontent.com/7679720/103656024-b5f83f80-4f2d-11eb-9a2d-fff8efa72a06.png)

Reply:
I agree that it would be nice to have it render correctly in both, but my primary concern is the docs site. Instead of using `<code></code>`, are there any obvious or alternative approaches you have in mind that do not break the inline HTML concern/linting rules?

Reply:
@IEvangelist Happy new year to you too!

I'd probably ask @DavidAnson for help on this. I opened https://github.com/DavidAnson/markdownlint/issues/363 asking for help on this.

Meanwhile, you can merge to fix the docs site, which is more important than GitHub-view.